conf/app.routes

```diff
@@ -98,3 +98,8 @@ GET        /check-your-answers                          controllers.CheckYourAns
 POST       /check-your-answers                          controllers.CheckYourAnswersController.onSubmit()
 
 GET        /confirmation                                controllers.ConfirmationController.onPageLoad()
+
+GET        /uploadWrittenAuthorisation                        controllers.UploadWrittenAuthorisationController.onPageLoad(mode: Mode = NormalMode)
```

Comment:
can you move these routes above, in order on how the happen in the journey?

I would leave here at the bottom the endpoints that happen at the end of the journey: confirmation and declaration.

It will be easier to read and maintain